README.md

```diff
@@ -1,13 +1,18 @@
-# Project Todos
+Project done during the Technigo Bootcamp
 
-Replace this readme with your own information about your project.
+# Vacay Planner Project
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The project was to do a todo web app using Redux, where you can add, delete and set tasks as done and not done.
+I turned it into a vacation planner.
```

Comment:
Such a nice idea with a vacation planner! 👍

Reply:
YES 😻 I haven't got a vacation for over 5 years! Maybe next year 😸